### PR TITLE
Update UTA data

### PR DESCRIPTION
**Pull request by ZachBeck**
refs https://github.com/agrc/porter/issues/113 and https://github.com/agrc/porter/issues/114

### CODE REVIEW

File: data/transportation/transit/index.html [hunk 1]
```diff
@@ -54,24 +60,40 @@
...
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="UTAStopsAndMostRecentRidership">UTA Stops & Most Recent Ridership</h3>
+  </div>
+  <div class="grid__col grid__col--12-of-12 package-content">
+    <img class="inline-text-left outline" src="{% link images/BusStops_Small.png %}" alt="Bus Stops" loading="lazy" />
+    <p>All UTA stops, including bus, Trax, and Frontrunner rail. Stop locations include Stop Name, City, Zip Code, County, UTA Stop ID and the Latitude and Longitude of the stop. Each stop displays Average Weekday Boarding and Alighting and which Routes are served at each stop. In UTA Stop ID: Numeric stops are regular bus stops, "BB" is Bus Bay, "FR" is a FrontRunner Station, "S" is Ski, "TX" is TRAX Station. This data is updated biannually.</p>
+    <p>{% include contact.html subject=page.title contact=site.data.contacts.uta %}</p>
+    <h5>Related Resources</h5>
+    <ul class="dotless">
+      <li><a href="https://opendata.gis.utah.gov/datasets/rideuta::uta-routes-most-recent-ridership">UTA Routes & Most Recent Ridership</a></li>
```
Comment (steveoh) [inline data/transportation/transit/index.html:89]: **question**: is this linking to the above layer? if so i think it is unnecessary.

File: data/transportation/transit/index.html [hunk 2]
```diff
@@ -12,12 +12,18 @@
   - uta
 categories: []
 date: 2011-11-23 14:16:26 -0700
-UTABusRoutesStops:
+UTARoutesAndMostRecentRidership:
   hub:
-    name: Utah UTA Bus Routes Wasatch Front
-    item_id: e5ba4be5e29040989ff50dc49ef0cc17
+    name: UTA Routes & Most Recent Ridership
+    item_id: 4347f3565fbe4d5dbb97b016768b8907
```
Comment (steveoh) [inline data/transportation/transit/index.html:18]: **issue**: this generates the wrong link. if it's shared you need to add the sharing org name.

File: data/transportation/transit/index.html [hunk 3]
```diff
@@ -12,12 +12,18 @@
...
   updates:
-    - April, 2020
+    - February, 2021
+UTAStopsAndMostRecentRidership:
+  hub:
+    name: UTA Stops & Most Recent Ridership
+    item_id: 41a38bb40ca245fdbfcc69ab8e50d80f
```
Comment (steveoh) [inline data/transportation/transit/index.html:24]: **issue**: this generates the wrong link. if it's shared you need to add the sharing org name.

**Review by steveoh — CHANGES_REQUESTED** [review]
i think these additions have quite a few issues. Please check out all the comments and let me know when they are resolved.

File: data/transportation/transit/index.html [hunk 4]
```diff
@@ -54,24 +60,40 @@
...
+    info=page.UTARoutesAndMostRecentRidership
+  %}
+</div>
+<div class="grid package">
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="UTAStopsAndMostRecentRidership">UTA Stops & Most Recent Ridership</h3>
+  </div>
+  <div class="grid__col grid__col--12-of-12 package-content">
+    <img class="inline-text-left outline" src="{% link images/BusStops_Small.png %}" alt="Bus Stops" loading="lazy" />
+    <p>All UTA stops, including bus, Trax, and Frontrunner rail. Stop locations include Stop Name, City, Zip Code, County, UTA Stop ID and the Latitude and Longitude of the stop. Each stop displays Average Weekday Boarding and Alighting and which Routes are served at each stop. In UTA Stop ID: Numeric stops are regular bus stops, "BB" is Bus Bay, "FR" is a FrontRunner Station, "S" is Ski, "TX" is TRAX Station. This data is updated biannually.</p>
```
[the remark this answers is not in the document]
Comment (ZachBeck) [reply data/transportation/transit/index.html:85]: I'll check with UTA and see what happened to the non numeric stop ID's. It think latitude/longitude vs X/Y is splitting hairs.

File: data/transportation/transit/index.html (hunk 2 above)
Comment (ZachBeck) [reply data/transportation/transit/index.html:18]: Where does the org name come from? Should the org name be, UTA or rideuta?

Comment (steveoh) [reply data/transportation/transit/index.html:18]: The org comes from their agol org name - e.g. `https://rideuta.maps.arcgis.com/`. Their org would be `rideuta`.

File: data/transportation/transit/index.html (hunk 3 above)
Comment (ZachBeck) [reply data/transportation/transit/index.html:25]: org added

File: data/transportation/transit/index.html (hunk 2 above)
Comment (ZachBeck) [reply data/transportation/transit/index.html:18]: org added

File: data/transportation/transit/index.html (hunk 1 above)
Comment (ZachBeck) [reply data/transportation/transit/index.html]: removed redundant link

File: data/transportation/transit/index.html [hunk 5]
```diff
@@ -54,24 +60,40 @@
...
   <div class="grid__col grid__col--12-of-12">
-    <h3 id="UTABusRoutesStops">UTA Bus Routes & Stops</h3>
+    <h3 id="UTARoutesAndMostRecentRidership">UTA Routes & Most Recent Ridership</h3>
   </div>
   <div class="grid__col grid__col--12-of-12 package-content">
     <img class="inline-text-left outline" src="{% link images/BusStops_Small.png %}" alt="Bus Stops" loading="lazy" />
-    <p>Utah UTA Bus Routes Wasatch Front includes bus stops along streets, at TRAX stations, and park & ride lots serviced by UTA buses. Utah UTA Bus Stops Wasatch Front includes redesigned bus routes that have been adjusted for internal monitoring purposes and ski bus routes to the Cottonwood Canyons and Sundance Ski Resort.</p>
-    <p>The UTA Bus Stops dataset was digitized to the Wasatch Front counties' street base maps and then exported from Trapeze (scheduling software) in shapefile format.</p>
+    <p>UTA bus, Trax, and Frontrunner routes and ridership values along the Wasatch Front. Route data includes route number (UTA's short name), route name, frequency in minutes, average ridership, and all cities and counties the route services. This data is updated biannually</p>
```
[the remark this answers is not in the document]
Comment (ZachBeck) [reply data/transportation/transit/index.html]: fixed

File: data/transportation/transit/index.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (ZachBeck) [reply data/transportation/transit/index.html]: fixed

**Review by steveoh — APPROVED** [review]

**Pull request merged**